services/training-service/README.md

```
@@ -0,0 +1,468 @@
+curl -X POST http://localhost:8000/sessions/my-session/sample \
+  -H "Content-Type: application/json" \
+  -d '{
+    "prompt": "The quick brown",
+    "max_tokens": 20,
+    "temperature": 0.7
+  }'
+```
+
+## Architecture
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)